app/src/main/java/org/wikipedia/activity/SingleWebViewActivity.kt

```diff
@@ -115,6 +116,7 @@ class SingleWebViewActivity : BaseActivity() {
     }
 
     private fun goBack() {
+        DonorExperienceEvent.logAction("article_return_click", "webpay_processed")
```

Comment:
This should have a logic to check whether the session is for donor experience. 

Can we add a parameter to the `newIntent()` such as `newIntent(logDonorExperience: Boolean = false)`?